content/_index.zh.html

```diff
@@ -0,0 +1,224 @@
+      </div>
+    </div>
+  </section>
+
+  <div class="spacer zero">
+    <a href="#welcome">
+      <img src="/images/homepage/down_arrow_special.svg" alt="">
+    </a>
+  </div>
+  <section class="welcome" id="welcome">
+    <h1>欢迎来到NavCoin</h1>
+    <div class="grid-container ninety-vw-container">
+      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <ul class="text-container">
+        <li class="">
+          <img src="/images/homepage/hom-fast-cheap.svg" alt="">
+          <h2>低摩擦支付</h2>
+          <p>专为快速、实惠和安全的对等支付而设计。一种旨在降低经营成本&nbsp;的&nbsp;网络。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-idea.svg" alt="">
+          <h2>2014年开始社区运营</h2>
+          <p>去中心化不只是一个流行词。它建立在网络的基础之上，创造一种未经许可的&nbsp;i创新&nbsp;文化。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-cfund.svg" alt="">
+          <h2>一种为自己买单的经济&nbsp;体系。</h2>
+          <p>NavCoin本身使用的是有史以来最先进的社区基金。第一个完全去中心化、完全自治、只有社区才能&nbsp;控制的&nbsp;基金。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-friendly.svg" alt="">
+          <h2>人性化的钱</h2>
+          <p>加密货币很复杂，但它不需要如此。NavCoin的目标是让你的父母用密码支付变得&nbsp;非常&nbsp;容易。</p>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <section class="features home-bg-gradient">
+    <div class="spacer one">
+      <div></div>
+    </div>
+    <div class="padding">
+      <h1>NavCoin的高级功能</h1>
+      <div class="center">
+          <div class="grid-container ninety-vw-container">
+            <div class="div-1">
+              <img src="/images/homepage/hom-low-energy.svg" alt="">
+              <h2>可持续股权证明</h2>
+              <p>NavCoin使用的是股权证明，消耗的能量只相当于工作网络证明的&nbsp;一&nbsp;小部分。</p>
+            </div>
+            <div class="div-2">
+              <img src="/images/homepage/hom-scalable.svg" alt="">
+              <h2>网络可伸缩性</h2>
+              <p>测试显示当前的处理能力为每秒260多个交易，同时积极研究第二层的扩展&nbsp;解决&nbsp;方案。</p>
+            </div>
+            <div class="div-3">
+              <img src="/images/homepage/hom-international.svg" alt="">
+              <h2>低成本全球化支付</h2>
+              <p>把钱发送到世界上任何地方都很便宜，不需要&nbsp;经过&nbsp;银行。</p>
+            </div>
+            <div class="div-4">
+              <img src="/images/homepage/hom-privacy.svg" alt="">
+              <h2>隐私</h2>
+              <p>从最初的NavTech到后来的升级，隐私一直是NavCoin的&nbsp;核心&nbsp;部分。</p>
+            </div>
+            <div class="div-5">
+              <img src="/images/homepage/hom-fast.svg" alt="">
+              <h2>快速支付</h2>
+              <p>交易在几秒钟内进行，30秒内得到确认，这对业务来说&nbsp;是&nbsp;理想的。</p>
+            </div>
+            <div class="div-6">
+              <img src="/images/homepage/hom-oa.svg" alt="">
+              <h2>支付一个名字，而不是一个数字</h2>
+              <p>NavCoin使用OpenAlias，发送钱就像输入电子邮件地址&nbsp;一样&nbsp;简单。</p>
```

Comment:
This section sounds like it has been google translated or needs improving

Reply:
fixed